platforms/qrb2210-rb1/partitions.conf

```diff
@@ -49,7 +49,8 @@
 --partition --name=dsp_b --size=65536KB --type-guid=77036CD4-03D5-42BB-8ED1-37E5A88BAA34
 --partition --name=abl_a --size=1024KB --type-guid=BD6928A1-4CE0-A038-4F3A-1495E3EDDFFB --filename=abl.elf
 --partition --name=abl_b --size=1024KB --type-guid=3D3E3AD2-8FF3-4975-A7E7-0E8A10B69F0D --filename=abl.elf
---partition --name=ddr --size=1024KB --type-guid=20A0C19C-286A-42FA-9CE7-F64C3226A794
```

Comment:
Does DDR partition describe memory / memory layout or something else? Why do we need two partitions here?

Reply:
The DDR partition usage on Agatti is to save DDR training data on the very first boot such that on subsequent boots the data can be restored from that partition avoiding to retrain DDR in order to optimize boot time.

The second slot is needed for DDR here since it is a requirement for boot team to use the same slot for all the images. I think it's more or less an anti-bricking requirement.